go.mod

```diff
@@ -12,6 +12,8 @@ require (
 	github.com/ethereum/go-ethereum v1.8.27
 	github.com/go-ldap/ldap/v3 v3.1.5
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
Why?

Reply:
This one keeps appearing automatically, I'll remove it.